stix/incident/__init__.py

```diff
@@ -454,6 +456,27 @@ def related_packages(self, value):
     def add_related_package(self, value):
         self.related_packages.append(value)
 
+    @property
+    def contacts(self):
+        return self._contacts
+
+    @contacts.setter
+    def contacts(self, contacts_list):
+        self._contacts = contacts_list
```

Comment:
`_informationSources` isn't a public data type (It had no associated type in STIX), so users aren't going to pass one in. Instead, they'll likely pass in instances of iterable collections like a `list` or `tuple`.

The `to_obj()`, `from_obj()`, `to_dict()`, and `from_dict()` methods all expect an `_InformationSources` object so we need to convert it to one in here (much like the `@coordinators.setter` method further down the module.

```python
self._contacts = _InformationSources(contact_list)
```

That will convert single instances or collections of of `InformationSource` objects into an `_InformationSources` object.